src/mumble/Settings.h

```diff
@@ -290,6 +290,15 @@ struct Settings {
 	QString qsProxyHost, qsProxyUsername, qsProxyPassword;
 	unsigned short usProxyPort;
 
+	/// iMaxInFlightTCPPings specifies the maximum
+	/// number of ping messages that the client has
+	/// sent, but not yet recieved a response for
+	/// from the server. This value is checked when
+	/// the client sends its next ping message. If
```

Comment:
Hmm. But we don't *have* non-periodic pings. I'm not sure what you're asking for here. s/pings/periodic pings/? Or just clarify that the feature is about the client-initiated TCP pings that happen every ~5s?